tests/utils/dev-server.js

```diff
@@ -7,12 +7,12 @@ const pidtree = require('pidtree')
 
 // each process gets a starting port based on the pid
 const rng = seedrandom(`${process.pid}`)
-function getRandomPortStart(rng) {
+function getRandomPortStart() {
   const startPort = Math.floor(rng() * 10000) + 10000 // 10000 to avoid collisions with frameworks ports
   return startPort
 }
 
-let currentPort = getRandomPortStart(rng)
```

Comment:
This is taken from the top-level scope, so it does not need to be passed as argument.